frontends/p4/simplifyDefUse.cpp

```diff
@@ -572,6 +580,7 @@ class FindUninitialized : public Inspector {
         headerDefs->clear();
         initHeaderParams(parser->getApplyMethodType()->parameters);
         visitVirtualMethods(parser->parserLocals);
+        unreachable = false;
```

Comment:
this is the fix; this code is similar to the handling of controls.
If there are virtual methods they may leave "unreachable" on true.